src/supervised/distance.jl

```diff
@@ -145,29 +153,43 @@ isstronglyconvex(::PeriodicLoss) = false
 # ===========================================================
 # L(y, t) = max(0, |y - t| - ɛ)
 
-immutable L1EpsilonInsLoss <: DistanceLoss
-    eps::Float64
+immutable L1EpsilonInsLoss{T<:AbstractFloat} <: DistanceLoss
+    ε::T
```

Comment:
Personally I think we should avoid unicode characters in anything "userfacing". But I could be convinced. Thoughts @tbreloff @joshday ?